api/tests/tooling/domain-builder/factory/build-campaign.js

```diff
@@ -102,12 +103,14 @@ buildCampaign.ofTypeProfilesCollection = function({
     customLandingPageText,
     archivedAt,
     type,
+    title: null,
     createdAt,
     creatorId,
     creator,
     organizationId,
     isRestricted,
-    organizationLogoUrl
+    organizationLogoUrl,
+    targetProfileId,
```

Comment:
```suggestion
    targetProfileId: null,
```
ce sera toujours le cas, il n'y a pas de profil cible dans une campagne de collecte de profils

Reply:
j'ai été obligé de l'ajouter car Bookshelf ne se comporte plus exactement pareil et retournait désormais `null` au lieu de `undefined` (ou l'inverse, je ne me rappelle plus exactement)

Reply:
j'ai rajouté un commit pour corriger ça ;)